docs/SOP.md

```diff
@@ -100,6 +101,17 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+## Reviving obsolete, orphaned, or inactive ontologies
+Ontologies that became dormant and were marked as either obsolete, orphaned, or inactive require a review process if they want to be marked as active again.
+Currently, these are the required steps:
+1. If an ontology is desired to be reactivated, a member of the ontology community first makes a PR on the metadata file with the desired change in state.
+This PR must include some evidence that backs up the change (we will provide better guidelines as this SOP matures). Think of showing closed issues to indicate activity etc.
+2. If the community member is not the ontology owner, the ontology owner must be contacted for comment via email and tagged on the issue. The ontology owner is given 1 month to object to the change, in which case, it is rejected.
+3. The OBO Operations committee assigns a reviewer during the next call to analyse the evidence for the change. Only the evidence matters - no need to try and collect more evidence in favour of the state change. Note that the ontology will be required to pass the Dashboard.
```

Comment:
ontology should be submitted to NOR dashboard and passing the NOR dashboard before ever being considered by the OBO Operations Committee, this is also the case for new ontology requests. 

I have prepared the testing suite to be extensible to applying certain tests in the inactive->active scenario or orphaned->active scenario, and will volunteer to make a technical solution to ensure that passing NOR dashboard is the case (if that's in line with what's been decided on, as I think this point states)

Reply:
i added that note about the dashboard (the rest is nico's text). i added it here because it is implicitly included in the review. i'll move it to #1.

Reply:
We have now decided against applying a specific test suite when moving an ontology to `active`, and instead defining new status codes for the case where ontologies don't pass. 

There is no need for us @cthoyt to die on this hill because starting 01.01.2024, passing the dashboard is generally mandatory, and so we can really be patient until then.

Reply:
sounds good, I don't think I was aware of the new rules about passing the dashboard when I wrote this initially, so this is actually even better than I could have hoped!

Reply:
Don't get too excited, because the war room is still in session about the meaning of the word "mandatory", like what kinds of consequences will it have to not pass.. Right now it's just a pretty vague commitment that was voted in. This is what we need to work on, a set of consequences that OBO ops will pass: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2360